scripts/jQuery.leanModal2.js

```diff
@@ -41,7 +43,11 @@
 
 					// IFHREF Fetch the Modal_ID
 					if ( $(this).attr('href') ) {
-						var modal_id = $(this).attr('href');
+						var modal_id = $(this).attr('href');						
+						//some times browsers add '/' character in the begining of the href attr 
+ 						if (modal_id.substring(0, 1) == '/') { 
```

Comment:
It turns out thee changes under this `if` are unnecessary. See my comment here: https://github.com/eustasy/jQuery.leanModal2/pull/4#issuecomment-180841177